cms/templates/index.html

```diff
@@ -296,7 +320,7 @@ <h3 class="course-title">${course_info['display_name']}</h3>
       </ul>
       % endif
 
-      %if len(courses) > 0:
+      %if len(courses) > 0 or optimization_enabled:
```

Comment:
If the optimization is enabled, I don't want to show the messaging about "create your first content library" because it implies that no libraries exist (which is not true on prod-- it's just that you haven't checked the box to show them).